packages/rum-core/src/performance-monitoring/breakdown.js

```diff
@@ -32,9 +32,12 @@ import { PAGE_LOAD } from '../common/constants'
  * Interested events from the Navigation timing API
  */
 const pageLoadBreakdowns = [
-  ['fetchStart', 'responseEnd', 'Network'],
-  ['domLoading', 'domInteractive', 'DOM Processing'],
-  ['domInteractive', 'loadEventEnd', 'Page Render']
+  ['domainLookupStart', 'domainLookupEnd', 'DNS'],
+  ['connectStart', 'connectEnd', 'TCP'],
+  ['requestStart', 'responseStart', 'Request'],
+  ['responseStart', 'responseEnd', 'Response'],
+  ['domLoading', 'domComplete', 'Processing'],
+  ['loadEventStart', 'loadEventEnd', 'Load']
```

Comment:
I think `Load event` is more clear here.

Reply:
Kinda agree with you, but it's lengthy. `Load` or `Onload` is bit small. Plus we have explanation docs for them, so we don't need to be more explicit here.

Reply:
@jahtalab do you think "Load" is unclear? I suppose "Load Event" would be more in line with the spec, but this seems clear enough to me.

Reply:
I think "Load" is an overloaded :) term. Personally if I see "Load" in another UI, I would be thinking which "Load" is it about, since there are multiple parts of a page loading process that can be referred to as "Load".

Reply:
Can we rename it to `Onload` ?

Reply:
we decided to use `Load` for now, we can change it later if necessary!